### PR TITLE
[GDI32_VISTA] Move D3DKMT functions out of gdi32. CORE-17433

### PR DESCRIPTION
**Pull request by ThFabba**
JIRA issue: [CORE-17433](https://jira.reactos.org/browse/CORE-17433)

### CODE REVIEW

**Review by learn-more — APPROVED** [review]

File: win32ss/gdi/gdi32_vista/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (JoachimHenze) [inline win32ss/gdi/gdi32_vista/CMakeLists.txt:1]: Line 1 contains 1 EOL whitespace.

Comment (JoachimHenze) [reply win32ss/gdi/gdi32_vista/CMakeLists.txt:1]: image

**Comment by JoachimHenze** [comment]
Would it make sense to create a base_address as well for that new gdi32_vista.dll? Or do you want to leave that for later?

**Comment by HBelusca** [comment]
image
🤔

File: win32ss/CMakeLists.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@ if(USE_DIBLIB)
 endif()
 
 add_subdirectory(gdi/gdi32)
+add_subdirectory(gdi/gdi32_vista)
```
Comment (SergeGautherie) [inline win32ss/CMakeLists.txt:14]: Would we want `gdi/gdi32/gdi32_vista`, like `dll/win32/kernel32/kernel32_vista`? (And unlike `dll/win32/advapi32_vista`..)

**Review by HBelusca — APPROVED** [review]

File: win32ss/gdi/gdi32_vista/CMakeLists.txt (hunk 1 above)
Comment (JoachimHenze) [reply win32ss/gdi/gdi32_vista/CMakeLists.txt]: okay, fixed when I retested 2021-12-29

**Pull request merged**